clean_project/clean_file_test.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+import tempfile
+import unittest
+
+
```

Comment:
you should remove a blank line here (only one blank line is suggested between system and local imports)